### ISSUE TITLE
Updated Field Names for Participant Mapping

### ISSUE THREAD

**Issue by SarahCooperNDI** [nditech/apollo#680]
The expected field names when mapping participant sets should match the way the columns are named in Excel when a participant set is exported. At the moment there are some inconsistencies (see below screenshot and attached Sample Participant Set Export).

![Apollo - Participant Set Fields to Map](https://user-images.githubusercontent.com/48285413/72821312-56caed80-3c3e-11ea-8a6b-6bd52cc485d5.jpg)

I would suggest we standardize so that in both places we use Participant ID (not ID) Location Code (not Location ID) and Supervisor ID (not Supervisor).

We should also drop the fields First Name, Last Name and Other Names as fields that we expect to map. We only ever have one name field in our participant files so Full Name covers all the bases.

[Sample Participant Set Export (CoCEG Guinea Instance).xlsx](https://github.com/nditech/apollo/files/4092036/Sample.Participant.Set.Export.CoCEG.Guinea.Instance.xlsx)

@blynchNDI @sarahmoulton @RLK-NDI

**Comment by takinbo** [comment]
I agree we should standardize column names all over the application. As regards dropping First Name, Last Name and Other Names, this feature was added as part of the requirements for integration with external CRM systems. Please see #240
